### PR TITLE
Added script/instructions to execute and process data in batch.

### PR DESCRIPTION
**Pull request by nilyibo**

### CODE REVIEW

File: analysis/results/iso_distributions/iso_distribution.all.csv [hunk 1]
```diff
@@ -0,0 +1,59 @@
+cloud,region,iso
```
Comment (actomlinson) [inline analysis/results/iso_distributions/iso_distribution.all.csv:1]: Do the files in `results/iso_distributions` take a long time to generate? If not, we probably don't need to check them in.

Also, it seems redundant to have each cloud in a separate file and also a combined file. Can we just use `iso_distribution.all.csv`? 

Lastly, we also need to have some kind of ground truth for the physical location (probably in lat/lon or a city name) of each datacenter. I'm wondering if we should have another csv with that information or if we can combine it into these csvs.

File: analysis/run_all.itdk_links.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# export SRC_REGION="TBD"
+export HOSTNAME="$(hostname -s)"
+
+# Note: these are meant to be run over multiple console windows and over multiple machines.
+#   Remove the tee redirects if you want to run them in the background.
+#   Use other batch execution systems if you want to automatically run them on multiple machines.
+
+# From AWS
+for SRC_REGION in $(echo $AWS_REGIONS); do
+	NUMANODE=0 && numactl --cpunodebind=$NUMANODE --membind=$NUMANODE /usr/bin/time -v ./itdk_links.py --src-cloud gcloud --src-regions $SRC_REGION --dst-cloud aws --dst-regions $(echo "$AWS_REGIONS") 1> >(tee $HOSTNAME.numa$NUMANODE.routes.aws.$SRC_REGION.aws.all.by_ip) 2> >(tee $HOSTNAME.numa$NUMANODE.routes.aws.$SRC_REGION.aws.all.err >&2)
```
Comment (actomlinson) [inline analysis/run_all.itdk_links.sh]: Should these be `--src-cloud aws` not `--src-cloud gcloud`?

Maybe define lines 16-20 as a function, and export a SRC and DST variable? Then you can call the function twice (one with SRC=AWS, DST=GCLOUD, and vice versa).

**Review by actomlinson — APPROVED** [review]

File: analysis/results/iso_distributions/iso_distribution.all.csv (hunk 1 above)
Comment (nilyibo) [reply analysis/results/iso_distributions/iso_distribution.all.csv:1]: No, they don't, but the results need manual inspection and format change into CSVs and they will almost never change, so I'm checking in the file to facilitate automated scripting of the entire process.

For redundancy, that is correct. I initially had separate files, but later on I need a unified file, so I can delete the two separate ones, but keeping the combined CSV will make it easier to run the later steps unmodified as in the README.

That's a good point. I think we can use the same file (with a different name), or different files (`cloud,region,gps`). Let me raise another PR when that's ready.

File: analysis/run_all.itdk_links.sh (hunk 2 above)
Comment (nilyibo) [reply analysis/run_all.itdk_links.sh]: Oh, good catch. Let me fix that. (This is not the actual script I ran :-( )

**Pull request merged**